NorthstarDLL/filesystem.h

```diff
@@ -5,25 +5,28 @@
 typedef void* FileHandle_t;
 
 #pragma pack(push, 1)
-struct VPKFileEntry
+
+// clang-format off
+OFFSET_STRUCT(VPKFileEntry)
 {
-	char* directory;
-	char* filename;
-	char* extension;
-	unsigned char unknown[0x38];
+	STRUCT_SIZE(0x44);
+	FIELD(0x0, char* directory)
+	FIELD(0x4, char* filename)
```

Comment:
4 bytes for a ptr?

Reply:
honestly i don't see much of a reason to explicitly define field offsets here, maybe a macro to define multiple fields inline from an offset (0 in this case) could be useful in situations where you don't need to do this? (assuming it isn't possible with the preexisting one

Reply:
This is fair actually. I could revert this to its old def